src/navigate-pop/tiling-stacking-windows.md

```diff
@@ -0,0 +1,107 @@
+# Manipulating Windows with Tiling & Stacking
+
+This section describes the Auto Tiling, Stacking, and window adjustment features in Pop!\_OS.
+
+---
+## Window Adjustment Mode
+
+Adjustment mode allows users to easily manipulate the placement of windows using keyboard shortcuts.
+
+Place a window into adjustment mode by bringing the window into focus, and then pressing `SUPER` + `Enter`. The window will be highlighted in yellow. You can now manipulate the window with the `🠐` `🠑` `🠒` `🠓` keys in combination with other keys.
+
+<video autoplay loop>
+    <source src="/images/tiling-stacking-windows/adjustment-mode-manipulation.webm" />
```

Comment:
This video looks great, but this is showing resizing with Shift, not movement with arrow keys. I would just show movement until you actually talk about the Shift key, otherwise you're just showing off a feature the user hasn't read about yet.

Regarding the resizing, it doesn't look like that's actually covered anywhere on this page; it probably should be. (And this initial Window Adjustment Mode section could reflect non-tiling mode since you haven't gotten to auto tiling yet.)